.github/workflows/website.yml

```diff
@@ -22,7 +22,9 @@ jobs:
           node-version: 16
       - uses: bahmutov/npm-install@v1
       - run: yarn test:website
+      - run: yarn build:website
```

Comment:
Motivation: https://filecoinproject.slack.com/archives/C02BZPRS9HP/p1648068210775589?thread_ts=1648065445.982479&cid=C02BZPRS9HP

(this will fail faster than the status quo, which is to only fail downstream)

Reply:
Build before test?

Reply:
FYI cloudflare builds the site for the preview link and it will fail there also. However the logs for that build are not available to community contributors so we're surfacing the error here also.

@gobengo lets avoid linking to the private slack channel from PRs as context gets lost for folks without access 🙏

Reply:
i pointed to this in slack but alan is right better to but it before test

Reply:
> lets avoid linking to the private slack channel from PRs as context gets lost for folks without access 

@alanshaw - sure thing. forgot it was private!

Reply:
moved build before test in https://github.com/nftstorage/nft.storage/pull/1698/commits/ea179157a5d0a70102e1bb70ecfc37857dde0fcc